src/internal/git/checkout.go

```diff
@@ -29,13 +29,13 @@ func CheckoutTagAsBranch(path string, tag string, branch plumbing.ReferenceName)
 	if err != nil {
 		message.Fatal(err, "Failed to locate tag in repository.")
 	}
-	checkoutHashAsBranch(path, tagRef.Hash(), branch)
+	CheckoutHashAsBranch(path, tagRef.Hash(), branch)
 }
 
-// checkoutHashAsBranch performs a `git checkout` of the commit hash associated
+// CheckoutHashAsBranch performs a `git checkout` of the commit hash associated
 // with the provided hash
 // It will delete the branch provided if it exists
-func checkoutHashAsBranch(path string, hash plumbing.Hash, branch plumbing.ReferenceName) {
+func CheckoutHashAsBranch(path string, hash plumbing.Hash, branch plumbing.ReferenceName) {
```

Comment:
This function was made public but nothing outside of this package is calling it?

Reply:
Was playing around with this to much and erroneously "standardized" it will fix all of these.

Reply:
Go is pretty unique where variables/functions that start with a lower case are 'package private' and things that start with an uppercase are 'public'

Reply:
The classic trap of, well maybe there is another reason the world is this way...